examples/Reranking_search_results_with_cross-encoders.ipynb

```diff
@@ -0,0 +1,805 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "7f30b8b2",
+   "metadata": {},
+   "source": [
+    "# Reranking search results with cross-encoders\n",
```

Comment:
small suggestion: could name it Search reranking with cross-encoders, just so it shows up next to the other search examples alphabetically

(fine either way)